### PR TITLE
Jarvis Profile Review 

### PR DESCRIPTION
**Pull request by bmaqs**
Please review my jarvis profile. I have edited this according to previous feedback. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,43 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Braikhna Yousafzai . Jarvis Consulting
 
+I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to understand my projects and company comprehensively. I appreciate team-based approaches towards a cohesive solution. I am a recent graduate from Western University with a Bachelor's degree with Specialization in Computer Science. I have worked on several Java, Python, and database management projects throughout my degree. I most enjoy the engineering, data management, and analytic aspects of the industry. I enjoy painting, coffee, and a good book outside of work. I am also quite involved with my community in volunteering and leadership capacities.
```
Comment (jarviscanada) [inline README.md]: I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to comprehensively understand my projects and company. I appreciate team-based approaches towards a cohesive solution. I am a recent graduate from Western University with a Bachelor's degree with Specialization in Computer Science. I have worked on several Java, Python, and database management projects throughout my degree. I most enjoy the software engineering, data management, and analytic aspects of the industry. I enjoy painting, coffee, and a good book outside of work. I am also quite involved with my community in volunteering and leadership capacities.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database. The data was collected in real-time using a `crontab` job and persisted using PostgreSQL. SQL queries were run to perform analytics on the cluster such as creating groups of nodes and determining average memory consumption. The project was deployed using a docker container, and version control was maintained through GitHub.
```
Comment (jarviscanada) [inline README.md]: Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database which was provisioned using Docker. The data was collected in real-time using a `crontab` job and persisted using PostgreSQL . Developed SQL queries to perform data analytics for cluster resource management purposes, such as determining average CPU and memory consumption and detecting node failures. Git and Github were used for version control.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+
+
+## Highlighted Projects
+**COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: Programmed a COVID-19 data tracker using C++ to allow for visual representations of worldwide COVID-19 trends. The project pulls COVID-19 data from API and the `iterator` object and `sort` function are used to parse and sort the data. The data is then graphed according to user input. This app issues graphical representation of trends such as countries with highest total case counts and countries with highest total death counts.
```
Comment (jarviscanada) [inline README.md]: **COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: Programmed a COVID-19 data tracker using C++ to allow for visual representations of worldwide COVID-19 trends. The project pulls COVID-19 data from API and the `iterator` object and `sort` function are used to parse and sort the data. The data is then graphed according to user input. This app issues a graphical representation of trends such as countries with the highest total case counts and countries with the highest total death counts.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,43 @@
...
+
+**COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Programmed a software allowing businesses to track incidences of COVID-19 in their establishments and automate exposure notifications to their customer base. Implemented customer, business, and app admin interfaces to separate role-based privileges. Businesses were able to send out exposure notifications to customer groups, while customers were able to receive notifications and report COVID-19 cases. The data was persisted on an online database server and SQL queries were used to select customer groups affected by potential exposures. To notify customers of exposures emails were sent out via a pipe. A GUI was designed using the Qt toolkit. This project was a collaborative submission for an Object Oriented Programming course submission. The project lifecycle was managed using agile and scrum frameworks.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Worked on development and engineering projects in an agile enviornment. Worked on a scrum team, including taking on duties as a scrum master.
```
Comment (jarviscanada) [inline README.md]: Worked on development and engineering projects in an agile environment. Worked on a scrum team, including taking on duties as a scrum master.

Responsibilities
Agile
Soft skills

File: README.md [hunk 5]
```diff
@@ -1,8 +1,43 @@
...
+
+**Data Analyst, First Canadian Title (2017-2019)**: Collaborated with the project team in the design, implementation, and deployment of a new vendor software. Administrated and managed the vendor database. Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to achieve performance and financial goals.
```
Comment (jarviscanada) [inline README.md]: Collaborated with the project team in the design, implementation, and deployment of a new vendor software. Administrated and managed the vendor database. Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to achieve performance and financial goals.

**Review by jarviscanada — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (jarviscanada) [reply README.md]: https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

File: README.md (hunk 1 above)
Comment (bmaqs) [reply README.md]: Updated to include corrections

File: README.md (hunk 2 above)
Comment (bmaqs) [reply README.md]: Updated to include these corrections

File: README.md (hunk 3 above)
Comment (bmaqs) [reply README.md]: Re worked this description to include more technical aspects of this project.

File: README.md (hunk 4 above)
Comment (bmaqs) [reply README.md]: Reworked this description to include more information

File: README.md (hunk 5 above)
Comment (bmaqs) [reply README.md]: Reworked this slightly to highlight more of my roles in the project.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,42 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Braikhna Yousafzai . Jarvis Consulting
 
+I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to comprehensively understand my projects and company. I appreciate team-based approaches towards a cohesive solution. I am a recent graduate from Western University with a Bachelor's degree with Specialization in Computer Science. I have worked on several Java, Python, and database management projects throughout my degree. I most enjoy the software engineering, data management, and analytic aspects of the industry. I enjoy painting, coffee, and a good book outside of work. I am also quite involved with my community in volunteering and leadership capacities.
```
Comment (jarviscanada) [inline README.md]: I am a recent graduate from Western University with a Bachelor's degree with a Specialization in Computer Science. I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to comprehensively understand my projects and company. I appreciate team-based approaches towards a cohesive solution. I have worked on Java, Python, and database management projects throughout my degree. I most enjoy the software engineering, data management, and analytic aspects of the industry. I enjoy painting, coffee, and a good book outside of work. I am also quite involved with my community in volunteering and leadership capacities.

File: README.md [hunk 7]
```diff
@@ -1,8 +1,42 @@
...
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database. The instance of PostgreSQL was provisioned using Docker. The data was collected in real-time using a `crontab` job. Developed SQL queries to perform data analytics for cluster resource management purposes such as determining average CPU and memory consumption and detecting node failures. Git and Github were used for version control.
```
Comment (jarviscanada) [inline README.md]: **Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database. The instance of PostgreSQL was provisioned using Docker. The data was collected in real-time using a `crontab` job. Developed SQL queries to perform data analytics for cluster resource management purposes, such as determining average CPU and memory consumption and detecting node failures. Git and Github were used for version control.

File: README.md [hunk 8]
```diff
@@ -1,8 +1,42 @@
...
+
+
+## Highlighted Projects
+**COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: Programmed a C++ utility to track worldwide COVID-19 trends. COVID-19 data was pulled from an API as CSV file and converted to a JSON file using jq as a JSON processor. The app was developed using the Visual Studio IDE and relied on C++ STL packages for algorithms and iterators, as well as file and I/O stream libraries. The application sorts the data and plots terminal-based bar graphs of the data. Users use CLI input to request graphs related to specific trends such as countries with highest total case and countries with highest total death counts.
```
Comment (jarviscanada) [inline README.md]: **COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: Programmed a C++ utility to track worldwide COVID-19 trends. COVID-19 data was pulled from an API as a CSV file and converted to a JSON file using the Linux jq command as a JSON processor. The app was developed using the Visual Studio IDE and relied on C++ STL packages for algorithms and iterators and file and I/O stream libraries. The application sorts the data and plots terminal-based bar graphs of the data. Users use CLI input to request graphs related to specific trends, such as countries with the highest total case and countries with the highest total death counts.

File: README.md [hunk 9]
```diff
@@ -1,8 +1,42 @@
...
+
+**COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Programmed a C++ based software allowing businesses to track incidences of COVID-19 in their establishments and automate email notifcations to a customer base. The application was designed using a factory buidler design pattern. An online SQL database was provisioned to host the data and SQL queries were used to select customer groups affected by potential exposures. The program connected to the database using the MySQL Connector for C++ library, which gave access to drivers and connectors for database access. Email notifications were sent out using pipes. Finally a GUI was designed using the Qt toolkit. Separate interfaces were built to isolate role-based privileges between customers, businesses, and admin. This project was a collaborative submission for an Object Oriented programming course submission. The project lifecycle was managed using agile and scrum frameworks.
```
Comment (jarviscanada) [inline README.md]: **COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Programmed a C++ based software allowing businesses to track incidences of COVID-19 in their establishments and automate email notifications to a customer base. The application was designed using a factory builder design pattern. An online SQL database was provisioned to host the data, and SQL queries were used to select customer groups affected by potential exposures. The program connected to the database using the MySQL Connector for C++ library, which gave access to drivers and connectors for database access. Email notifications were sent out using pipes. Finally, a GUI was designed using the Qt toolkit. Separate interfaces were built to isolate role-based privileges between customers, businesses, and admin. This project was a collaborative submission for an Object-Oriented programming course submission. The project lifecycle was managed using agile and scrum frameworks.

File: README.md [hunk 10]
```diff
@@ -1,8 +1,42 @@
...
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Collaborated with developer, data engineers, and product managers in an agile enviornment. Developed projects using technologies and toolkits such as Java, SQL, Docker, Linux, Bash, PostgreSQL, and Maven. Applied Gitflow as an organizational practice to project development. Participated in scrum ceremonies, including daily scrums, sprint retrospectives, and code reviews. Also took on responsibilities as a scrum master, heading the sprint retrospectives and providing resource planning to my team.
```
Comment (jarviscanada) [inline README.md]: **Software Developer, Jarvis (2020-present)**: Collaborated with developers, data engineers, and product managers in an agile environment. Developed projects using technologies and toolkits such as Java, SQL, Docker, Linux, Bash, PostgreSQL, and Maven. Applied Gitflow as an organizational practice to project development. Participated in scrum ceremonies, including daily scrums, sprint retrospectives, and code reviews. Also took on responsibilities as a scrum master, heading the sprint retrospectives and providing resource planning to my team.

File: README.md [hunk 11]
```diff
@@ -1,8 +1,42 @@
...
+
+**Data Analyst, First Canadian Title (2017-2019)**: Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to acheive performance and financial goals. Collaborated with the project team in the UI design and deployment of a new vendor software. Created user stories, worked with the QA team to create and run test cases, as well as compiled a user manual for end users.
```
Comment (jarviscanada) [inline README.md]: **Data Analyst, First Canadian Title (2017-2019)**: Built a reporting suite using SQL to visualize vendor performance and client KPI's. Worked as a vendor advisor and client solution consultant finding innovative strategies to achieve performance and financial goals. Collaborated with the project team in the UI design and deployment of new vendor software. Created user stories, worked with the QA team to develop and execute test cases, and compiled a user manual for end-users.

File: README.md [hunk 12]
```diff
@@ -1,8 +1,42 @@
...
+
+
+## Education
+**Western University (2018-2021)**, Bachelor of Science, Computer Science
+- Dean's List (2020, 2021)
+- GPA: 3.7/4.0
+
+
+## Miscellaneous
+- Art, painting: host fundraising paint nights at our local mosque.
```
Comment (jarviscanada) [inline README.md:41]: git rm -r profile/.idea

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**